### ISSUE TITLE
Provide a jump off point in a Show page to see other shows? 

### ISSUE THREAD

**Issue by orta** [artsy/Emergence#52]
On artsy.net for example you can see other shows from the same partner at the bottom: https://www.artsy.net/show/onishi-gallery-contemporary-japanese-kogei

**Comment by katarinabatina** [comment]
The idea was to allow users to flick into the next show once they reached the end of the install shots for a single show. Don't know if @owendodd has designs for this yet.

**Comment by orta** [comment]
oh, that's a nice idea :+1: 

so then swiping left when you start would move you back a show too?

**Comment by owendodd** [comment]
@orta yeah that was the original thought. Although I do also see the value in offering multiple exit points. Going to spend some time playing w/ the TV today and will circle back w/ more ideas
